ci/shellcheck.sh

```diff
@@ -6,7 +6,7 @@ set -e
 cd "$(dirname "$0")/.."
 (
   set -x
-  git ls-files -- '*.sh' ':(exclude)ci/semver_bash' \
```

Comment:
I think this one works.

I tried to make ci unhappy in https://github.com/solana-labs/solana/pull/31298/commits/a0fdfb40db8aa226a2669e474a06fb0f45f8c5c4 and the pipeline failed expectedly
<img width="694" alt="Screenshot 2023-04-23 at 2 08 44 PM" src="https://user-images.githubusercontent.com/8209234/233822957-9bbc7bc1-4c3a-4d23-9d76-c29287afceee.png">
(https://buildkite.com/solana-labs/solana/builds/94353#0187acb6-8f1e-4a3b-bf20-71924ad2a645)